src/diskquota.h

```diff
@@ -215,8 +215,9 @@ struct DiskquotaDBEntry
 	TimestampTz last_run_time;
 	int16       cost; // ms
 
-	bool inited; // this entry is inited, will set to true after the worker finish the frist run.
-	bool in_use; // this slot is in using. AKA dbid != 0
+	bool inited;    // this entry is inited, will set to true after the worker finish the frist run.
+	bool in_use;    // this slot is in using. AKA dbid != 0
+	bool corrupted; // consider this entry as invalid to start the worker on
```

Comment:
It seems that only one line was added here, but for some reason two have changed!

Reply:
Otherwise, CI check for code style fails:
git ls-files *.{c,h} | xargs clang-format -i --style=file && git diff --exit-code